src/amo/components/AddonsCard/index.js

```diff
@@ -91,6 +102,7 @@ export default class AddonsCard extends React.Component<Props> {
       type && `AddonsCard--${type}`);
     return (
       <CardList
+        key={`AddonsCard${this.getKey()}`}
```

Comment:
I don't see why we'd need to add a key here, or maybe I'm not understanding. I thought we only needed a key when we're asking react to render a number of items in a loop. Here we're only rendering a single `CardList` component, so why would we need a key?

Reply:
Right, you should definitely use in the looping array scenarios but I’ve also seen when react does unexpected things (like renaming a class to sibling's class - which is what it does here - see screenshots above), adding a key usually can help with this. Here are a few articles that discuss this to some degree. 

https://www.jstips.co/en/react/keys-in-children-components-are-important/

https://reactjs.org/docs/lists-and-keys.html

These article are talking more in the looping array sense (similar to what you mentioned above ) but they discuss react being able to differentiate between the siblings (which is what it seems to be having trouble doing here).  

This seems to be the problem code (from Addons file):
```javascript
<div className="Addon-main-content">
     {this.renderAddonsByAuthorsCard({ isForTheme: true })}
     …
     {this.renderShowMoreCard()}
     ...
</div>
```
For some reason react is not able to differentiate this for some reason when there is a description (which is what `renderShowMoreCard` renders) and adding a key helps. I think because they both have the `Card` component as (nested) child..

Maybe there is a another better way but as far as I understand adding a key doesn't hurt anything, it just helps react out with differentiating between siblings if needed.

note 1: I also noticed that If you add a div around either render function/or component show above it is another way to help with the issue... but I thought it was better to use the key approach so we are not changing the html structure.

note 2: I could also move adding the key to the `Card` component (vs `CardList`) since that is the component that is used by both `ShowMoreCard` and `AddonCard`